apps/web/src/pages/TrainingOpportunities/CreateTrainingOpportunityPage.tsx

```diff
@@ -87,16 +87,12 @@ const CreateTrainingOpportunityForm = ({
     <FormProvider {...methods}>
       <form onSubmit={handleSubmit(onSubmit)}>
         <Card>
-          <div
-            data-h2-display="base(flex)"
-            data-h2-justify-content="base(center) p-tablet(flex-start)"
-          >
+          <div className="xs:justify-flex-start flex justify-center">
             <Heading
               level="h2"
               color="secondary"
               icon={IdentificationIcon}
-              data-h2-margin="base(0, 0, x1.5, 0)"
-              data-h2-font-weight="base(400)"
+              className="mb-9 font-normal"
             >
```

Comment:
The wrapping div is no longer necessary. You can add the class names directly to the `Heading` like we did in other pages:

https://github.com/GCTC-NTGC/gc-digital-talent/blob/5a1b15734a5fb928b9275cfc2daa464d7d696f4f/apps/web/src/pages/WorkStreams/CreateWorkStreamPage.tsx#L145-L152